modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase

```diff
@@ -1561,12 +1561,37 @@ definition {
 	}
 
 	@description = "Verify that the user can can publish a definition with Task node"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanPublishWithTaskNode {
+		property test.name.skip.portal.instance = "ProcessBuilderKaleoDesignerReact#CanPublishWithTaskNode";
+
+		Workflow.openWorkflowListView();
 
-		// TODO CanPublishWithTaskNode pending implementation
+		Workflow.addNewDefinition(workflowDefinitionTitle = "New Workflow Definition");
+
+		ProcessBuilderKaleoDesignerReact.dragSidebarNodeAndMove(
+			nodeLabel = "Task",
+			nodePosition = "0,0");
```

Comment:
Italo, the node position needs to be "0,0" or you could not pass, cause this make a confusion with the other nodes, i have made the changes on [another pull request](https://github.com/brianchandotcom/liferay-portal/pull/118297/files) and sent. Thanks!